app/controllers/concerns/api/v2/extracted_puppet_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Api
+  module V2
+    module ExtractedPuppetController
+      extend ActiveSupport::Concern
+
+      included do
+        prepend_before_action :fail_and_inform_about_plugin
+
+        resource_description do
+          desc 'This resource has been deprecated, to continue using it please install Foreman Puppet Enc plugin and use its API enpoints.'
```

Comment:
With a deprecation, you can continue using it. This is just removal.
```suggestion
          desc 'This resource has been removed. To continue using it please install the Foreman Puppet plugin and use its API endpoints.'
```

Reply:
I've reworded this, although we are using deprecated for other resources that were extracted, so it will not be consistent.
So someone should fix it there. Although it is ambigous here, you can continue using it, just need to install the plugin.